Amplify/Categories/Logging/Internal/LoggingCategory+CategoryConfigurable.swift

```diff
@@ -27,6 +27,11 @@ extension LoggingCategory: CategoryConfigurable {
 
         try plugin.configure(using: configuration?.plugins[plugin.key])
         self.plugins[plugin.key] = plugin
+        
+        if let consolePlugin = try? self.getPlugin(for: AWSUnifiedLoggingPlugin.key) {
```

Comment:
maybe i am out of loop, but why do we need a separate call to `configure()` here?

Reply:
we need to support both cloudwatch and and console plugin, I'm forcing a configure on the console logging so that it reads the new logging configuration file to check whether console is enabled or not.

Reply:
got it